crates/wac-resolver/Cargo.toml

```diff
@@ -38,7 +39,7 @@ tokio-util = "0.7.10"
 tempdir = "0.3.7"
 
 [features]
-default = ["registry"]
+default = []
```

Comment:
Unless it makes it harder to implement, I'd rather we leave the registry feature in in this PR. We can discuss it's removal either in an issue or in a PR specific to its removal.